### PR TITLE
OLM 16.1.0

### LINKED ISSUE #425: OLM structure for 16.1.0
### Description

Create OLM structure for 16.1.0 to be published on operatorhub

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by jonathanvila**
## Related GH Issue
Closes #425 

## Additional Information
New 16.1.0 of the Keycloak needs a new Operator

## Test Steps
Deploy / Upgrade the operator from the Openshift Marketplace ( test catalog )
Check that it starts and the console opens fine.

## Checklist:
- [X] Automated Tests
- [ ] [Documentation](https://github.com/keycloak/keycloak-documentation) changes if necessary
- [ ] (If required) Discussed on [Keycloak DEV](https://groups.google.com/forum/#!forum/keycloak-dev)

### CODE REVIEW

**Comment by jonathanvila** [comment]
Community-operators PR : https://github.com/k8s-operatorhub/community-operators/pull/547
Comuunity-operators-prod PR : https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/574

**Review by vmuzikar — APPROVED** [review]

File: test/e2e/keycloaks_test.go [hunk 1]
```diff
@@ -311,7 +311,7 @@ func prepareKeycloaksCR(t *testing.T, f *framework.Framework, ctx *framework.Con
 
 func prepareKeycloaksCRWithExtension(t *testing.T, f *framework.Framework, ctx *framework.Context, namespace string) error {
 	keycloakCR := getKeycloakCR(namespace)
-	keycloakCR.Spec.Extensions = []string{"https://github.com/aerogear/keycloak-metrics-spi/releases/download/1.0.4/keycloak-metrics-spi-1.0.4.jar"}
+	keycloakCR.Spec.Extensions = []string{"https://github.com/aerogear/keycloak-metrics-spi/releases/download/2.5.3/keycloak-metrics-spi-2.5.3.jar"}
```
Comment (vmuzikar) [inline test/e2e/keycloaks_test.go:314]: Could you please remove this change? It is already addressed by #432.

**Review by vmuzikar — CHANGES_REQUESTED** [review]

**Comment by vmuzikar** [comment]
@jonathanvila bump :)

File: test/e2e/keycloaks_test.go (hunk 1 above)
Comment (jonathanvila) [reply test/e2e/keycloaks_test.go:314]: :+1:

**Review by vmuzikar — APPROVED** [review]

**Pull request merged**